compiler/one-cmds/tests/one-infer_006.py

```diff
@@ -0,0 +1,14 @@
+# This script gets one argument and print it
+
+import sys
+from pathlib import Path
```

Comment:
I think this file is excluded in format checker.
@yunjayh , can you also look at format checker, if not a huge burden. if not I'll check.